README.md

```diff
@@ -2,30 +2,69 @@
 
 # LogDNA CloudWatch
 
-The LogDNA AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.com/documentation/lambda/) to route your [CloudWatch Logs](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html) to LogDNA.
+The LogDNA AWS CloudWatch integration relies on [AWS Lambda](https://aws.amazon.com/documentation/lambda/) to route your [CloudWatch logs](http://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html) to LogDNA.
 
 ## Configure the LogDNA AWS Lambda function
 
-1. Create a new Lambda function and skip to the Configure function section
-2. Set the following fields:
-* Name: `LogDNA`
-* Runtime: `Python 2.7`
-* Code entry type: `Upload a .ZIP file`
-* Upload a [.ZIP file](https://s3.amazonaws.com/repo.logdna.com/integrations/cloudwatch/lambda.zip)
-* Handler: `logdna_cloudwatch.lambda_handler`
-* Environment variables: 
-    * (Required) LOGDNA_KEY: LOGDNA_KEY YOUR_INGESTION_KEY_HERE
-    * (Optional) LOGDNA_HOSTNAME: Alternative Host Name
-    * (Optional) LOGDNA_TAGS: Comma-separated Tags
-    * (Optional) LOGDNA_URL: Alternative LogDNA Server
-4. Click `Create function`
-
-## Configure your AWS CloudWatch Stream
-
-1. Select the CloudWatch Log Group to upload to LogDNA
-2. Click the Actions menu and select `Stream to AWS Lambda`
-3. Select the `LogDNA` Lambda function and click `Next`
-4. Select the desired log format and click `Start Streaming`
+1. Create a [new Lambda function](https://console.aws.amazon.com/lambda) and select `Author from scratch`
+2. For the basic information:
+ * Function Name: `logdna_cloudwatch` (you can choose what to name it)
+ * Runtime: `Node.js.10.x` (`Python 2.7` if using in `Python`)
```

Comment:
You seem to be using Python 3 in the CircleCI configs above - is Python 2.7 here correct?

Reply:
Oooops... need to change CircleCI one